examples/package.json

```diff
@@ -15,7 +15,7 @@
   "license": "Apache-2.0",
   "main": "./dist/cjs/index.js",
   "module": "./dist/esm/index.js",
-  "types": "./dist/esm/index.d.ts",
+  "types": "./dist/cjs/index.d.ts",
```

Comment:
what's the reason for this change?

Reply:
matches the rest now but 

had some issue, tried setting the types to point to a diff path, not work, find and replace back, also standardize the one I missed earlier